docs/sources/_index.md

```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
 
 We write our documentation using U.S. English.
 
+Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in the Writers' Toolkit, refer to Google's style guide.
+
+If you're new to the guide, start with the [Get started]({{< relref "./get-started" >}}) section.
+
```

Comment:
I think the flow of this introduction section could be improved, but I'm not sure how to go about it.